amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2540,16 +2542,18 @@ paths:
           description: Current user does not have permission to create retention schedule
         '404':
           description: recordCategoryId does not exist
+        '409':
+          description: Retention schedule already exist for the given recordCategoryId
         default:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
     get:
       tags:
         - retention-schedule
-      summary: Get a list of retention schedules
+      summary: Get the retention schedule for a record category
```

Comment:
Even if we change the API back to be plural then I would leave this documentation - it's much clearer knowing that there can only be one schedule per category.

Reply:
Agreed. kept the documentation as it is.